css-conditional-4/Overview.bs

```diff
@@ -27,6 +27,28 @@ and have not yet been copied here.
 
 Issue: In the future, copy the contents of [[css3-conditional]] into this document.
 
+<h2 id="at-supports-ext">Extensions to the ''@supports'' rule</h2>
+
+This level of the specification extends the <<supports-feature>> syntax as follows:
+
+<pre class="prod def" nohighlight>
+	<dfn>&lt;supports-feature></dfn> = <<supports-selector-fn>> | <<supports-decl>>
```

Comment:
Shouldn’t `<<supports-decl>>` be first?

Reply:
Doesn't really matter since they cannot conflict, right?

Reply:
They shouldn’t, but I prefer it to be alphabetically sorted and/or appended because it looks nicer.